java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2077_java.html

```diff
@@ -102,4 +102,6 @@ <h2>See</h2>
   (JPA)</a>, <a href="http://h3xstream.github.io/find-sec-bugs/bugs.htm#SQL_INJECTION_JDO">Potential SQL/JDOQL Injection (JDO)</a>, <a
   href="http://h3xstream.github.io/find-sec-bugs/bugs.htm#SQL_INJECTION_HIBERNATE">Potential SQL/HQL Injection (Hibernate)</a> </li>
 </ul>
+<h2>Deprecated</h2>
+<p>This rule is deprecated; use {rule:squid:S3649} instead.</p>
```

Comment:
I wonder will this link work if security rules are not available (as in community edition)

Reply:
Indeed it will link to the empty page in community edition. @agigleux should we do something?